<launch>
  <node name="vision_filter" pkg="project" type="filtered_vision_follow.py"/>
  <node name="position_plotter" pkg="project" type="plotter.py"/>
  <node name="simple_follow" pkg="project" type="simple_follow.py"/>
</launch>
